english/car/regera.txt

```diff
@@ -0,0 +1 @@
+An hybrid supercar by Koenigsegg. 0-400km/h in 20 holy seconds! and a top speed of 410. Four cars sold before anybody saw the car.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.